runtime/src/main/java/org/corfudb/runtime/view/stream/StreamAddressSpace.java

```diff
@@ -111,6 +111,16 @@ public void addAddress(long address) {
         addressMap.addLong(address);
     }
 
+    /**
+     * Add an array of addresses to the stream address space
+     * @param addresses add addresses to the underlying bitmap
+     */
+    public void addAddresses(long ... addresses) {
```

Comment:
Looks like it is used by a unit test only, do we need it? If so, the underlying bitmap has API to add an array. `addressMap.add(addresses)`